tests/src/rules/no-unused-modules.js

```diff
@@ -1374,7 +1374,10 @@ describe('parser ignores prefixes like BOM and hashbang', () => {
   });
 });
 
-describe('supports flat eslint', { skip: !FlatRuleTester }, () => {
+describe('supports flat eslint', function() {
+  if (!FlatRuleTester) {
+    return this.skip();
+  }
```

Comment:
these two should be equivalent, based on how javascript and mocha work :-/

Reply:
and the failures are because `this.skip` isn't a function, which is also weird.

Reply:
The syntax that was there does look familiar, but fwiw I don't see it documented in mocha anymore. I guess I missed that `this.skip` is not available in `describe`. I just did an early return instead. The test is running now, but it passes so I'm back where I started. Although, I don't know if it matters, but the test fails when run by itself.

Reply:
ah ok, if that’s the problem we can fix it with `(FlatRuleTester ? describe : describe.skip)(`